packages/nx-plugin/src/generators/page/schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "cli": "nx",
+  "$id": "page",
+  "title": "Create an Analog page",
+  "type": "object",
+  "properties": {
+    "name": {
```

Comment:
I think this should be `path` or `pathname`. It represents the URL path to the page.